.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,37 @@
+name: Bug Report
+description: File a bug report
+labels: ["bug"]
+body:
+  - type: checkboxes
+    attributes:
+      label: Has the bug already been reported?
+      description: "Please check that the bug you're reporting hasn't already been reported. You can use the GitHub issues search box to help with this."
```

Comment:
I'm not sure if there's some way for us to link to the issues page here (we can't hard-code the URL since it's different for reach repo)